### ISSUE TITLE
Select LABEL => exception in console

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#779]
@peller, @jhpedemonte, @billreed63

Assigned to me to try to reproduce. Maybe the same as #770

With M2 on staging server, I had two theme editors open, app.css open and an HTML file open when I did these steps:
- New HTML file (this is now the 5th open editor)
- Add a LABEL
- Click away from LABEL
- Click on LABEL to select

```
TypeError: Result of expression 'this.visualEditor.context' [undefined] is not an object.
```

**Comment by peller** [comment]
dup of #770

**Comment by peller** [comment]
oops... that checkin comment should have referenced #790 for the editor.fileName fix, but the latter part applies to #770, which I do believe is a dup of this.  @JonFerraiolo, please retest.  I can't reproduce this now.

**Comment by JonFerraiolo** [comment]
Cannot reproduce. Probably fixed as part of #770 fix. Closing.
